packages/maps/src/components/result/GoogleMapMarkers.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import types from '@appbaseio/reactivecore/lib/utils/types';
+
+import GoogleMapMarker from './GoogleMapMarker';
+
+const GoogleMapMarkers = ({ showMarkers, resultsToRender, ...rest }) => (
+	<div>
+		{showMarkers
```

Comment:
How about checking this condition in the parent component?  In this way, we can stop the extra div to render

Reply:
Sure thing 👍